src/core/scope.ml

```diff
@@ -253,6 +256,15 @@ let scope : mode -> sig_state -> env -> p_term -> tbox = fun md ss env t ->
         find_qid false d.m_rhs_prv ss env qid
     | (P_Iden(qid,_)   , _                 ) ->
         find_qid false false ss env qid
+    | (P_Wild, M_URHS(data)) ->
+      let x =
+        let name = Printf.sprintf "v%i" data.m_urhs_vars_nb in
+        let x = Bindlib.new_var te_mkfree name in
+        data.m_urhs_vars_nb <- data.m_urhs_vars_nb + 1;
+        data.m_urhs_xvars <- (name, x) :: data.m_urhs_xvars;
+        x
+      in
+      _TEnv (Bindlib.box_var x) [||]
```

Comment:
Why is the array argument empty? You should take the environment into account. After the doc: "_ for an unknown term or a term we don’t care about. It is replaced by a fresh metavariable (or a fresh pattern variable in a rule left-hand side) applied to all the variables of the context." For instance, in `\x:N,\Pi y:_,a`, _ is equivalent to `&Y[x]` where Y is fresh.

Reply:
Indeed it's a mistake.